lib/jekyll-last-modified-at/git.rb

```diff
@@ -8,6 +8,7 @@ class Git
       def initialize(site_source)
         @site_source = site_source
         @is_git_repo = nil
+        @lcd_cache = {}
```

Comment:
What does "lcd" stand for here? As opposed to what other kind of cache?

Reply:
I reckon it stands for "last commit date", per the next hunk